### DISCUSSION TITLE
Fix task list formatting

### DISCUSSION THREAD

**Discussion by txtyash** [remarkjs/remark#998] [General] [answered]
Why does remark-ls escape brackets in task lists? Can we please fix task list formatting in remark?

**Comment by wooorm** [comment]
What is remark-ls?

**Reply by ChristianMurphy** [reply]
I'm assuming https://github.com/remarkjs/remark-language-server

**Comment by ChristianMurphy** [comment]
@zim0369 how do you have remark configured?
Do you have `remark-gfm` as part of your configuration? (as task lists are a part of that standard)

**Comment by txtyash** [comment]
@ChristianMurphy I did not configure it. I installed it with [Lunarvim](https://github.com/LunarVim/LunarVim) with the `LspInstall` command. I'm not sure what plugin provides that command but the plugin comes preinstalled with [Lunarvim](https://github.com/LunarVim/LunarVim).

**Comment by ChristianMurphy** [comment, marked as answer]
> I'm not sure what plugin provides that command but the plugin comes preinstalled with [Lunarvim](https://github.com/LunarVim/LunarVim).

I'm not sure I follow, I don't see any reference to the remark in Lunarvim.
If it does come pre-installed and pre-configured by Lunarvim, that would likely be a question for them.

If it is in fact using vanilla (un-configured) remark-language-server.
The behavior you are seeing is expected, the language server defaults to commonmark, which does not include checklists.
If you want to add checklist support, add a `.remarkrc` file, and include `remark-gfm` as a plugin (https://github.com/unifiedjs/unified-engine/blob/main/doc/configure.md), this will enable GitHub flavored markdown which includes checklist support.
